public/apps/configuration/panels/role-view/index-permission-panel.tsx

```diff
@@ -66,7 +66,7 @@ export function renderFieldLevelSecurity() {
       return (
         <EuiFlexGroup direction="column" style={{ margin: '1px' }}>
           <EuiText key={'-'} size="xs">
-            -
+            &mdash;
```

Comment:
Non-blocking: I would suggest we define a shared constant since it appears a number of times.

Reply:
Added shared constant